test/cqlpy/test_guardrail_compact_storage.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from cassandra.protocol import InvalidRequest
+
+from .util import config_value_context, new_test_table
+
+# Tests for the enable_create_table_with_compact_storage guardrail.
+# Because this feature does not exist in Cassandra , *all* tests in this file are
+# Scylla-only. Let's mark them all scylla_only with an autouse fixture:
+@pytest.fixture(scope="function", autouse=True)
+def all_tests_are_scylla_only(scylla_only):
+    pass
```

Comment:
Nitpick (feel free to ignore, it's not important):
If you have a file with just two tests, it's simpler just to add scylla_only on those two places and not use this autouse and 5-line comment explaining it :-(

A different question for a different time (not important now) is why you even have a test file with just one test ;-)